### ISSUE TITLE
Add all fields for Pipedrive connector

### ISSUE THREAD

**Issue by impowski** [airbytehq/airbyte#6918]
## Tell us about the problem you're trying to solve

We are trying to export all of the possible entity fields such as `DealFields`, `OrganizationFields`, etc.

## Describe the solution you’d like

Similar to `ActivityFields` retrieve all of the `Fields` entities

## Describe the alternative you’ve considered or used

I've seen that the raw data from Pipedrive has all of the custom fields for `Deals` and probably it would be possible to map them manually using custom transformer

## Are you willing to submit a PR?

Looking at the source code of the current Pipedrive `stream.py` file I think I can do it.

**Comment by AnatolePledg** [comment]
Hi @impowski 

How have you done this using DBT ? It seems to me that the data related to Fields isn't even retrieved from the API by the current version of the connector. To me it's not a matter of Transformation but a matter of Extraction.

Thanks

**Comment by impowski** [comment]
@AnatolePledg it's actually retrieving raw data which contains JSON objects which contain all available fields. You just need to select `Raw data` and add custom transformations. I can share an example if you want

**Comment by AnatolePledg** [comment]
Thanks @impowski ! I indeed saw the fields in the raw tables. Yes, please, if you can share the transformation that would help a lot, especially if you were able to bring in the field names.
Thanks

**Comment by impowski** [comment]
@AnatolePledg basically you need to follow the transformations guide and you will end up with a project. Where you'll do something like this in `ab1` file:
```
{{ json_extract_scalar('_airbyte_data', ['<your-field-token>'], ['<your-field-token>']) }} as name,
```

Then in `ab2` you will cast values it will be different for each type (this is in case of strings):
```
cast({{ empty_string_to_null('name') }} as {{ dbt_utils.type_string() }}) as name,
```

In `ab3` you'll just select those fields

**Comment by sergiworkwize** [comment]
How did you find the ab1 file for the deals stream? I get this following the transformation tutorial: 

![Screenshot 2022-01-20 164916](https://user-images.githubusercontent.com/97533688/150373043-9ba0050a-33c2-4f8b-ab3c-93fb42f45f9e.png)

So I only see the part based on the deals_ab3 file

**Comment by marcosmarxm** [comment]
Closed by #11870
